### PR TITLE
Increases tram collision damage

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request
Increases the damage caused from being hit by the tram. Now it'll give you some broken bones if the RNG isn't on your side.
## Why It's Good For The Game
Respect the tram. Grind the rails, but you're gonna maybe break a leg when it hits you.
## Changelog
:cl: LT3
balance: Getting hit by the tram may now break some of your bones, or more. Be careful!
/:cl:

### CODE REVIEW

**Comment by funman816** [comment]
Based!!!

**Comment by Rocko1345** [comment]
Always wanted this, to hear that satisfying crunch of bone when the tram hits you headon

File: code/modules/industrial_lift/industrial_lift.dm [hunk 1]
```diff
@@ -58,7 +58,7 @@ GLOBAL_LIST_EMPTY(lifts)
 	/// if TRUE, we will gib anyone we land on top of. if FALSE, we will just apply damage with a serious wound penalty.
 	var/violent_landing = TRUE
 	/// damage multiplier if a mob is hit by the lift while it is moving horizontally
-	var/collision_lethality = 1
+	var/collision_lethality = 2
```
Comment (Kylerace) [inline code/modules/industrial_lift/industrial_lift.dm:61]: shouldnt be used like this, this is meant to be a multiplier on the default damage of the tram. you should create defines for the lowest and largest base damage for the tram and increase those, then put them in /obj/structure/industrial_lift/proc/travel()

File: code/modules/industrial_lift/tram_lift_master.dm [hunk 2]
```diff
@@ -8,7 +8,7 @@
 	var/travel_distance = 0
 
 	///multiplier on how much damage/force the tram imparts on things it hits
-	var/collision_lethality = 1
+	var/collision_lethality = 2
```
Comment (Kylerace) [inline code/modules/industrial_lift/tram_lift_master.dm:11]: i forgot to actually do anything with the var on this datum, should just be removed

**Review by Kylerace — CHANGES_REQUESTED** [review]

File: code/modules/industrial_lift/industrial_lift.dm (hunk 1 above)
Comment (lessthnthree) [reply code/modules/industrial_lift/industrial_lift.dm:61]: We use collision_lethality during the tram malfunction event.

Comment (lessthnthree) [reply code/modules/industrial_lift/industrial_lift.dm:61]: But yeah I'll go edit base damage.

File: code/modules/industrial_lift/industrial_lift.dm [hunk 3]
```diff
@@ -389,7 +389,7 @@ GLOBAL_LIST_EMPTY(lifts)
 					continue
 				to_chat(collided, span_userdanger("[src] collides into you!"))
 				playsound(src, 'sound/effects/splat.ogg', 50, TRUE)
-				var/damage = rand(5, 10) * collision_lethality
+				var/damage = rand(9, 28) * collision_lethality
```
Comment (YakumoChen) [inline code/modules/industrial_lift/industrial_lift.dm:392]: doesn't this make the minimum damage less than before?

Comment (Kylerace) [reply code/modules/industrial_lift/industrial_lift.dm:392]: no

Comment (Kylerace) [reply code/modules/industrial_lift/industrial_lift.dm:392]: unless you mean before as in the pre review version of this pr, in which case yes

**Comment by Serxule** [comment]
tram is already decently painful and if you're unlucky it will crit you, now you'll probably just get crit normally and have to go to medical to fix your bones if you're unlucky, will make tram an even more disliked map

**Comment by CMDR-Gungnir** [comment]
> tram is already decently painful and if you're unlucky it will crit you, now you'll probably just get crit normally and have to go to medical to fix your bones if you're unlucky, will make tram an even more disliked map

Skill issue.

Memes aside, my dream for the tram is that getting hit just instantly kills you. Respect the tram. It's funny.

**Comment by Overuseddart** [comment]
Does this take armor into account? If I get hit as say, an elite and all my bones break I’m pretty sure I’d activate my bomb implant.

**Comment by YakumoChen** [comment]
> Does this take armor into account? If I get hit as say, an elite and all my bones break I’m pretty sure I’d activate my bomb implant.

skill issue

**Pull request merged**